bentoml/_internal/configuration/__init__.py

```diff
@@ -42,7 +42,7 @@ def expand_env_var(env_var):
 
 
 def is_pip_installed_bentoml():
-    is_installed_package = hasattr(version_mod, "version_json")
+    is_installed_package = len(version_mod.version_tuple) == 3
```

Comment:
we will start with pre-release on pypi which will break this, pre-release looks like `1.0.0.a2`

Reply:
:) it shouldn't

Reply:
tried building a pre-release with `1.0.0.a2` and `bentoml._version.version_tuple == (1,0,0)` after `pip install`, so it's working as expected.

I do run into another case, where after I tag the branch with any string in semver format, it will think it's a pypi released version, even if it's a editable install